src/main/java/org/spongepowered/api/CatalogType.java

```diff
@@ -35,7 +37,7 @@
  * <code>`a.getId().equalsIgnoreCase(b.getId())`</code> are true then all must
  * be true.</p>
  */
-public interface CatalogType {
+public interface CatalogType extends TextRepresentable {
```

Comment:
@ST-DDT: I agree with what @gabizou said. The problem with `Text` is that it usually implies some sort of formatting (color or otherwise). There's no real meaningful 'default' formatting that can be provided for the human-readable name of a `CatalogType`, so I think it makes sense to just keep `String getName()`.